Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -123,8 +123,35 @@ function validate_cards() {
         SetTestStateAborted
         return 1
     fi
+    echo "****************************" >> TestExecution.log
+    echo "Running xbutil scan" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
     xbutil scan >> TestExecution.log
+
```

Comment:
should we have fail/pass check after each xbutil command?

Reply:
I added the check for return value `$?` after each `xbutil` run